Gemfile.lock

```diff
@@ -55,52 +55,52 @@ GEM
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.1)
```

Comment:
### [activejob](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.0.2.2 → 6.0.3.1

[_[change-log](https://github.com/rails/rails/blob/v6.0.3.1/activejob/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.3.1/activejob)_]





<details>
<summary>Commits</summary>

A change of **257** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.2.2...v6.0.3.1).

These are the first 10 commits:
- (e5310b5) [Merge pull request #37849 from kamipo/fix_since_and_ago](https://github.com/rails/rails/commit/e5310b5d745b63ca6a344508390d9d3920dff379)
- (21a1b18) [Revert "registers previous commit in the CHANGELOG"](https://github.com/rails/rails/commit/21a1b187df474d8aa8f75d776bdeefd6c91a58f1)
- (67b7923) [Revert "let environments configure load-related paths"](https://github.com/rails/rails/commit/67b79238c4ab320427f03486c0b9b525f7b53e44)
- (dc84717) [Add file analysis to the Active Storage guide [ci skip]](https://github.com/rails/rails/commit/dc847176647a3f377ef4d3065921b82ca8019eea)
- (675f967) [Controller can be symbols as well](https://github.com/rails/rails/commit/675f967a314c3029137d1ed6a306c85d5a4a958f)
- (480745c) [Merge pull request #37862 from giraffate/fix_random_ci_failu…](https://github.com/rails/rails/commit/480745c56752fb381ba0b84b59eb58cec62eb72e)
- (596e125) [Merge pull request #37866 from giraffate/correct_default_val…](https://github.com/rails/rails/commit/596e1250cfde2768232432071365a10554517c9c)
- (24bf3f0) [Merge pull request #37882 from kamipo/should_keep_zero_durat…](https://github.com/rails/rails/commit/24bf3f08c096d1cbb934a0265cb707a8d3f9bfc2)
- (d2f53ac) [Switch to the writer DB to generate ASt previews](https://github.com/rails/rails/commit/d2f53ac5c306557c3d3204399b34fccfbd4d6baa)
- (e193acb) [Merge pull request #37666](https://github.com/rails/rails/commit/e193acb118b2bbcb4fa06d727f63a26e62405d9f)


</details>